docker/prod/Dockerfile

```diff
@@ -27,4 +27,3 @@ RUN chmod +x /app/wait-for-db.sh
 
 ENV NODE_ENV=production
 
-USER node
```

Comment:
@dwojno  Is this a temporary removal for the fix or is this permanent?

Reply:
This only temporary, cause there were an issue to write to file while being an user that is not a root ;)